src/main/java/io/aiven/kafka/connect/common/output/OutputWriter.java

```diff
@@ -43,6 +43,8 @@ public void writeRecord(final SinkRecord record) throws IOException {
         writeOneRecord(record);
     }
 
+    public void close()  throws IOException {
```

Comment:
The class should implement `AutoCloseable`, thus Java will be able to handle it nicely as other closable resources, particularly in the try-with-resources idiom.

Reply:
- [ ] use AutoClosable